src/main/java/de/zalando/tip/zalenium/servlet/LiveNodeHtmlRenderer.java

```diff
@@ -1,8 +1,8 @@
 package de.zalando.tip.zalenium.servlet;
 
 import com.google.gson.JsonObject;
+import de.zalando.tip.zalenium.proxy.DockerSeleniumRemoteProxy;
 import org.openqa.grid.common.exception.GridException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.grid.common.exception.GridException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)